mlir/utils/jenkins/Jenkinsfile.miopen-test

```diff
@@ -64,7 +64,7 @@ mlir/utils/jenkins/miopen-tests/**\
             matrix {
                 agent {
                     docker {
-                        image 'rocm/mlir:rocm4.1-latest'
+                        image 'rocm/mlir:latest'
```

Comment:
'rocm/mlir:rocm4.1-latest' should be equivalent with 'rocm/mlir:latest', since ROCm 4.1 is what we got.

Reply:
> maybe it's because we ROCm 4.2 has been released

As long as we haven't bumped to ROCm 4.2 in the dockerfile, it won't happen automatically.

I took a look a second ago and see `rocm/mlir:latest` and `rocm4.1-latest` having the exact same commit hash. Can you try to pull again, I suspect you have a out-dated image.

Reply:
I'm not sure how to get Jenkins to re-pull

Reply:
I did some googling and find [this image](https://i.stack.imgur.com/ZP0vN.png). Could you verify this? I think the idea is to force jenkins pull the latest docker instead of using the local out-of-dated cached image.

Reply:
We may not want to *always* have that on, but it's probably a good idea since we've recently made changes to our Docker images

Reply:
`alwaysPull true` does what we want - it always pulls, but `pull` itself won't redownload if nothing's changed

We may just want to check that we're not building new images too often.

Reply:
Thanks for confirming. 

> pull itself won't redownload if nothing's changed

That's pretty reasonable behavior.

> We may just want to check that we're not building new images too often.

Agreed, but I think we are already doing the best we can. Besides, the overhead of downloading a fresh image isn't that much so I won't worry about it now.